Source/WebCore/html/parser/HTMLPreloadScanner.cpp

```diff
@@ -515,7 +519,7 @@ void HTMLPreloadScanner::scan(HTMLResourcePreloader& preloader, Document& docume
 
     while (auto token = m_tokenizer.nextToken(m_source)) {
         if (token->type() == HTMLToken::Type::StartTag)
-            m_tokenizer.updateStateFor(AtomString(token->name()));
+            m_tokenizer.updateStateFor(AtomString::lookUp(token->name().data(), token->name().size()));
```

Comment:
`lookUp` is different from normal constructor since it can return null AtomString when we do not find the corresponding one in the table (so, it is just `lookUp`).

Reply:
Yes. The `updateStateFor` function only compares against known attribute names. So if it’s not already an existing atom string, it’s fine to pass null since it’s known to not be any of the known attribute names. I could also check for null and not call `updateStateFor` at all in that case.

Reply:
I’m going to leave this as-is.